front/components/AmountChooser.js

```diff
@@ -93,6 +93,9 @@ export default class AmountChooser extends React.Component {
                     <input type="checkbox" name="abonnement" id="donation-monthly"/>
                     <label htmlFor="donation-monthly">Je veux donner ce montant chaque mois</label>
                 </div>
+                <div className="amount-chooser__help">
```

Comment:
`class` instead of `className` no ?

Reply:
not in [JSX](https://reactjs.org/docs/jsx-in-depth.html)

Reply:
XD thanks!